wlog-glog/logger_test.go

```diff
@@ -201,6 +201,29 @@ func TestDiag1Log(t *testing.T) {
 	}
 }
 
+func TestWrapped1Audit2Log(t *testing.T) {
+	os.Args = []string{
+		os.Args[0],
+		"-logtostderr=true",
+	}
+	flag.Parse()
+
+	entityName := "entity"
+	entityVersion := "version"
+	for _, tc := range wrapped1logtests.Audit2TestCases(entityName, entityVersion) {
+		// TODO: test output
```

Comment:
seems like an important todo?

Reply:
there are TODOs on all of the tests for this implementation, would prefer not to block. Thoughts?

Reply:
Yeah alright, I see this is in glog where the testing is already bad. We should clean up these todos but probably not worth blocking on